src/dock_state/tree/mod.rs

```diff
@@ -180,25 +180,31 @@ impl<Tab> Tree<Tab> {
         })
     }
 
-    /// Returns the number of nodes in the `Tree`.
+    /// Returns the number of nodes in the [`Tree`].
+    /// 
+    /// This includes [`Empty`](crate::Node::Empty) nodes.
     #[inline(always)]
     pub fn len(&self) -> usize {
         self.tree.len()
     }
 
-    /// Returns `true` if the number of nodes in the tree is 0, `false` otherwise.
+    /// Returns [`true`](bool) if the number of nodes in the tree is 0, otherwise [`false`](bool).
```

Comment:
These links to `bool` are unnecessary, see how Rust's core library handles bools: https://doc.rust-lang.org/nightly/src/core/option.rs.html#602

Reply:
ah! ill get that removed then, https://github.com/Adanos020/egui_dock/pull/149/commits/95185afafc3ea44474e2b1c4ed811bfe7b6160cb